src/core/execute-serialize.c

```diff
@@ -2720,7 +2720,9 @@ static int exec_context_deserialize(ExecContext *c, FILE *f) {
                         if (r < 0)
                                 return r;
                 } else if ((val = startswith(l, "exec-context-private-tmp="))) {
-                        c->private_tmp = private_tmp_from_string(val);
+                        /* Before v257, the field was serialized with serialize_bool_elide(). Hence, "yes"
```

Comment:
No. There's no need. We pin the executor fd to guard against incompatible changes, which is completely different from how serialization of `Manager` and `Unit` objects are handled.

Reply:
`ExecContext` is _not_ serialized on reexec. We load the settings from unit fragments. This serialization format is solely used for sd-executor invocations.

Reply:
Indeed. I found the comment in execute-serialize.h. Thanks.

Reply:
The commit is dropped now.